### DISCUSSION TITLE
Updated to blank screen

### DISCUSSION THREAD

**Discussion by maromaro70** [HASwitchPlate/HASPone#16] [Q&A]
Connected a new plate (premade, connected to AC) to wifi and mqtt, went to web UI and updated from url, saved, the screen turned off and that's it! Waited over 20 minutes then recycled the power, no change, still at a blank screen. Any thoughts would be really appreciated.

**Comment by aderusha** [comment]
Is it connecting to WiFi still? Can you access the admin web page?

**Reply by maromaro70** [reply]
I can see it online, can't connect to it with web UI

**Reply by aderusha** [reply]
If this is a unit you bought from me, I can just send you a replacement with a label to return this one, so if that sounds like easy mode for you I start with that offer :)

If you want to troubleshoot, your next step will likely involve opening the device and connecting to the USB port buried inside.  That should be kicking out debug information via serial @ 115200 bps.  That should allow us to figure out WTF is going on.

**Reply by maromaro70** [reply]
Would it be ok to get the replacement please? 3 kids at home is not conducive to patient troubleshooting. I can't wait for schools to open again

**Reply by maromaro70** [reply]
Left the house for a couple hours, now it is flashing a "HASP initiating" message every second. I'll disconnect it and wait. Thank you for your help, Allen!

**Reply by maromaro70** [reply]
To try and make sure it is not a firmware issue, I tried flashing HASwitchPlate.tft using a microSD card, formatted with FAT32, no luck, the screen still simply flashes almost every second.

**Reply by aderusha** [reply]
> To try and make sure it is not a firmware issue, I tried flashing HASwitchPlate.tft using a microSD card, formatted with FAT32, no luck, the screen still simply flashes almost every second.

This sounds like the HASP firmware is stuck in a boot loop for some reason.  The Nextion defaults powered off, and early in the HASP boot process it lights up the pin which will power on the LCD.  So, something is crashing the sketch and that happens shortly after the call to turn on the LCD.  Serial debug output would likely tell you more (or you can just forget about it and I'll send you a replacement).

**Reply by maromaro70** [reply]
A replacement would be greatly appreciated, please. What's your preferred communication method? I might be interested in a second unit with single plate cover as well, if possible, or I can simply buy it online; just thinking you'll be shipping the replacement anyway.

**Reply by aderusha** [reply]
What would work best for me would be to have you order the unit, and I can then ship you two in a larger box along with a return label.  If you don’t think that you’re ready to buy another one straight away, that’s OK and I can just send the replacement, but it’d save me a bit on shipping to have it all bundled together.  All new units being shipped today will come with HASPone installed and ready for use.

**Reply by aderusha** [reply]
One more thing - if you do that, leave a note with your Github username in the order so I know what to do :)

**Reply by maromaro70** [reply]
:)

**Comment by IIAIronWolf** [comment]
That's really odd. I just did mine with no problem. Sounds like it messed up during the firmware loading. Have a power flash?

**Reply by maromaro70** [reply]
No visible power flash that I could see with lights on. Very odd, it booted just fine initially, connected to WiFi and HA MQTT broker with no issues until the update process started.
